webapp/src/selectors/index.tsx

```diff
@@ -5,3 +5,7 @@ const pluginPrefix = `plugins-${plugin_constants.pluginId}`;
 export const getprojectDetailsState = (state: any) => {
     return state[pluginPrefix].projectDetailsSlice;
 };
+
+export const getRhsState = (state: any): {isSidebarOpen: boolean} => {
```

Comment:
Why is the type of `state` `any` here?

Reply:
this was left for the enhancements part but as we are now doing that, fixed it now